ParallaxGenLib/src/ParallaxGenDirectory.cpp

```diff
@@ -155,13 +156,19 @@ auto ParallaxGenDirectory::mapFiles(const unordered_set<wstring> &NIFBlocklist,
       WinningSlot = NIFUtil::getSlotFromTexType(WinningType);
     }
 
+    bool IgnoreTexture = false;
+    if (isFileInBSA(Texture, BSAExcludes) && (WinningSlot == NIFUtil::TextureSlots::PARALLAX)) {
```

Comment:
These should probably be flipped so that it checked if it is parallax first, since that's faster than isFileInBSA, then it wouldn't need to check for a bunch